AutomatedTesting/Gem/PythonTests/Prefab/tests/create_prefab/CreatePrefab_UnderAnotherPrefab.py

```diff
@@ -51,7 +51,7 @@ def CreatePrefab_UnderAnotherPrefab():
     entity = inner_instance.get_direct_child_entities()[0]
     # We track if that is the same entity by checking the name and if it still contains the component that we created before
     assert entity.get_name() == "TestEntity", f"Entity name inside inner_prefab doesn't match the original name, original:'TestEntity' current:'{entity.get_name()}'"
-    assert entity.has_component("PhysX Collider"), "Entity name inside inner_prefab doesn't have the collider component it should"
+    assert entity.has_component("PhysX Primitive Collider"), "Entity name inside inner_prefab doesn't have the collider component it should"
```

Comment:
NIT: Why not import the PHYSX_PRIMITIVE_COLLIDER from Editor Python Test Tools so we can eliminate passing hand typed/pasted strings around while getting the benefit of autocomplete?

`from editor_python_test_tools.consts.physics import PHYSX_PRIMITIVE_COLLIDER`
`assert entity.has_component(PHYSX_PRIMITIVE_COLLIDER), "Entity name inside inner_prefab doesn't have the collider component it should"`

Reply:
Thanks! I'll clean this up in a follow-up PR.

Reply:
Follow up PR doing this clean up: https://github.com/o3de/o3de/pull/14955